src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -318,13 +317,33 @@ describe('ReactDOMComponent', () => {
       expect(container.firstChild.className).toEqual('');
     });
 
-    it('should properly update custom attributes on custom elements', () => {
+    it('should properly update custom properties on custom elements', () => {
```

Comment:
Note that when you change many native dom properties (such as `node.id` and `node.className`) that the corresponding attributes (`id` and `class`) are automatically changed to reflect the property value. So users will still be able to effectively set the `id` and `class` attributes (among others), by changing the `id` and `className` properties on the element.